tools/api-markdown-documenter/src/test/test-data/README.md

```diff
@@ -1,3 +1,8 @@
 Each directory under this folder represents an "API Model".
 Each contains 1 or more `.api.json` files representing individual packages within the model.
 End-to-end tests will point to individual model directories for documentation generation testing.
+
```

Comment:
I realized this wasn't documented anywhere. The setup here is a bit annoying, but seems better than actually checking in mock packages into the repo?